app/templates/board.html

```diff
@@ -1,6 +1,9 @@
 <div class="row" class="bz-board">
-    <button ng-click="ShowByAssignee = !ShowByAssignee" class = "btn btn-success" ng-hide="ShowByAssignee"> Currently showing columns by status. Click To Show by Assignee</button>
-    <button ng-click="ShowByAssignee = !ShowByAssignee" class = "btn btn-success" ng-show="ShowByAssignee">Currently showing columns by assignee. Click To Show by Status</button>
+    <div style="text-align: center;">
+        <button ng-click="ShowByAssignee = !ShowByAssignee" class = "btn btn-success" ng-hide="ShowByAssignee"> Currently Showing Columns By Attribute 'Status'. <br> Click To Show By 'Assignee'.</button>
```

Comment:
Please remove the classes, `class="..."` from these buttons. I think leaving the style of the button to bugzilla is the best option.